Package.swift

```diff
@@ -18,10 +18,10 @@ let package = Package(
   ],
   targets: [
     .plugin(
-      name: "AirbnbSwiftFormatPlugin",
+      name: "FormatSwift",
       capability: .command(
         intent: .custom(
-          verb: "format",
+          verb: "airbnb-format-swift",
```

Comment:
This change will break the Lottie repo since the repo depends on the master branch of this repo. I will put up a PR to Lottie before merging this change.

Reply:
Lottie is pinned to a specific commit in its `Package.resolved` right?

Reply:
We should likely start using releases in this repo too though once we make sure this plugin works effectively.

Reply:
Good call @calda on the `Package.resolved` 👍  I'm glad we are checking those files in. In any case, I've removed the change to this `verb` so it's a moot point. I agree with @fdiaz that we should start adding releases to this repo, and a changelog.

Reply:
Sounds good -- I suppose we'll make a new release when we make config changes from adding a new rule to the style guide?

Reply:
> Sounds good -- I suppose we'll make a new release when we make config changes from adding a new rule to the style guide?

👍

Reply:
To double check we understand how this works, I tested the Lottie repo after this change. Running `swift package format` used the previous commit as specified in `Package.resolved`. When I run `swift package update`, it updates `Package.resolved` to reference the most recent commit. I do agree that we should start cutting releases for this repo once we stabilize the tool though.

Reply:
Thanks for checking @calda